packages/object-code/src/hash.ts

```diff
@@ -27,6 +27,53 @@ import { sortNumbers } from './util';
 export function hash(val: unknown, seen?: WeakSet<object>): number {
   let h = 5381;
 
+  // Handle objects with entries() method but no enumerable keys
+  // (e.g., FormData, URLSearchParams, Map, Set)
+  if (
```

Comment:
revert back to the previous change you had, this is a lot of bloat.

I just asked you to grab the mapper function and move it to a helper


`something.map(a => b)`, so just move `a =>b ` to a halper function and `something.map(helperFn)`